src/Parse/Assetic/ScssCompiler.php

```diff
@@ -1,12 +1,11 @@
 <?php namespace October\Rain\Parse\Assetic;
 
 use Event;
```

Comment:
@bennothommo @JaxWilko @christianWilling is there value in moving this filter to the Assetic core? I'm not 100% sure what it does differently from the already included Assetic filters.

@bennothommo also note that the Event facade is included but not used here.